vms/platformvm/service.go

```diff
@@ -2015,32 +2032,50 @@
 	}
 
 	// Takes in byte repr. of a staker.
-	// Returns the amount staked that belongs to an address in [addrs]
-	helper := func(tx *Tx) (uint64, error) {
+	// Returns:
+	// 1) The total amount staked by addresses in [addrs]
+	// 2) A description of the staked outputs
+	helper := func(tx *Tx) (uint64, []StakedOut, error) {
```

Comment:
Should this live on a func of it's own ? Instead of being declared on each call ?